monitor.h

```diff
@@ -1,3 +1,11 @@
 #pragma once
 
+// Define constants for the upper and lower limits
+const double TEMPERATURE_LOWER_LIMIT = 95.0;

```

Comment:
Nice names :)
Can specify the unit where applicable. Temperature would be in Fahrenheit, while pulse and spo2 may be clear and don't need the units